azure-pipelines.yml

```diff
@@ -1,16 +1,30 @@
 variables:
-  Agent.Source.Git.ShallowFetchDepth: 1
   CI_NAME: Azure Pipelines
   CI_BUILD_ID: $(Build.BuildId)
   CI_BUILD_URL: "https://dev.azure.com/sunpy/sunpy/_build/results?buildId=$(Build.BuildId)"
+  CIBW_BUILD: cp36-* cp37-*
+  CIBW_SKIP: "*-win32 *-manylinux1_i686"
+  CIBW_TEST_COMMAND: pytest -p no:warnings --doctest-rst -m "not figure" --pyargs sunpy
+  CIBW_TEST_REQUIRES: sqlalchemy scikit-image glymur beautifulsoup4 drms python-dateutil zeep tqdm asdf hypothesis pytest-astropy pytest-mock
 
-jobs:
-- template: azure-templates.yml
-  parameters:
-    name: Linux_36_offline
-    os: linux
-    tox: py36-offline -- -n=4
+trigger:
+  batch: true
```

Comment:
What does this do?

Reply:
`If you set batch to true, when a build is running, the system waits until the build is completed`

Reply:
Could remove it.